Sources/SweetTableController.swift

```diff
@@ -24,6 +26,16 @@
             self.addConstraints()
         }
 
+        open override func viewWillAppear(_ animated: Bool) {
+            super.viewWillAppear(animated)
+
+            if self.clearsSelectionOnViewWillAppear {
```

Comment:
I don't follow. Isn't that what `clearsSelection...` does already? Also why is it defaulting to `false` now? 🤔